app/controllers/welcome_controller.rb

```diff
@@ -1,4 +1,15 @@
 class WelcomeController < ApplicationController
-    def index
+  def index
+  end
+
+  def quotes
```

Comment:
See my comment on your `routes.rb` file. Why did you chose to not be RESTful and create this action in the welcome controller vs creating a quotes_controller with an index action?